### ISSUE TITLE
Black on black in Linux

### ISSUE THREAD

**Issue by asdf23** [spesmilo/electrum#8938]
I'm using a dark theme on my OS (Fedora Wayland Gnome)
What I'm seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/196476b2-a4cf-4e5e-a80b-42925c00a67e)

What I'm _not_ seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/cf0e39fb-6055-4116-ac75-6dd059fcb27f)

Seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/eff8dd48-bee1-4c8b-8810-3d6667253609)

Not seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/92855b7f-b094-48ea-98f0-83e31f13d684)

**Comment by asdf23** [comment]
Seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/83d9544f-d80b-4a1c-958d-aa1552b92029)

Not seeing:
![image](https://github.com/spesmilo/electrum/assets/4165735/d080e62d-feff-47c7-b340-9296532ac8b7)

**Comment by asdf23** [comment]
This Stackoverflow aticle might be helpful: https://stackoverflow.com/questions/74991889/how-to-handle-system-dark-mode-in-qml 

tldr;
There are styles in QtQuick that are system theme aware by default. Material.theme: Material.System

Reference:
https://doc.qt.io/qt-6/qtquickcontrols-material.html#material-theme-attached-prop

Possible bug found here:  electrum/gui/qml/components/main.qml

```
....snip....
ApplicationWindow
{
    id: app

    visible: false // initial value

    // dimensions ignored on android
    width: 480
    height: 800

    Material.theme: Material.Dark
    Material.primary: Material.Indigo
    Material.accent: Material.LightBlue
    font.pixelSize: constants.fontSizeMedium
....snip....
```

**Comment by ecdsa** [comment]
The above file is for the QML GUI, which differs from the Qt GUI.
The screenshots you are showing are for the Qt GUI.

**Comment by asdf23** [comment]
Launching the program with this works:
`Exec=env GTK_THEME=adwaita:light QT_STYLE_OVERRIDE=Adwaita electrum`

**Comment by asdf23** [comment]
Or this desktop shortcut:

```
[Desktop Entry]
Comment=Lightweight Bitcoin Client
Exec=electrum %u
GenericName[en_US]=Bitcoin Wallet
GenericName=Bitcoin Wallet
Icon=electrum
Name[en_US]=Electrum Bitcoin Wallet
Name=Electrum Bitcoin Wallet
Categories=Finance;Network;
StartupNotify=true
Exec=env GTK_THEME=adwaita:light QT_STYLE_OVERRIDE=Adwaita electrum
Terminal=false
Type=Application
MimeType=x-scheme-handler/bitcoin;x-scheme-handler/lightning;
Actions=Testnet;

[Desktop Action Testnet]
Exec=electrum --testnet %u
Name=Testnet mode
```
